src/fiatExchanges/utils.tsx

```diff
@@ -34,7 +34,7 @@ interface ProviderRequestData {
   userLocation: UserLocationData
   walletAddress: string
   fiatCurrency: LocalCurrencyCode
-  digitalAsset: CiCoCurrency
+  digitalAsset: string
```

Comment:
why is this changing to string?

Reply:
The old values were CiCoCurrency matched up with the values that cloud-function's fetchProviders expects, but that is no longer the case

Reply:
shouldn't there be a type for what fetchProviders expects? So we can have compile-time checks for when there might be an error in a request to that cloud function?